config/Freso/clink.yaml

```diff
@@ -0,0 +1,6 @@
+nvchecker:
+  git: https://github.com/Lurk/clink.git
+  include_regex: ((?:\d|\.)+)
+  source: git
```

Comment:
Please use `source: github` if possible.

Reply:
I’ve switched to using `git` over `github` by request from @dvzrv: https://gitlab.archlinux.org/archlinux/packaging/packages/picard/-/merge_requests/1#note_176510 – I’m not really interested in maintaining different versions of nvchecker configuration between the AUR repositories and this one, so if this is a blocker, then I’ll remove the github ones from this PR.

Reply:
I've read `https://github.com/lilydjwg/nvchecker/blob/master/nvchecker_source/git.py` and I think it will be OK.